src/test/java/studytestsuitenew/Study_002_ItemsTests.java

```diff
@@ -0,0 +1,47 @@
+package studytestsuitenew;
+
+import businessobjects.Item;
+import org.testng.annotations.Test;
+import pageobjectsnew.CartPageNew;
+import pageobjectsnew.IndexPageNew;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+import static studytestsuitenew.HelpMethodsNew.studyTexts;
+
+/**
+ * Created by user on 06.06.17.
+ */
+public class Study_002_ItemsTests extends DefaultTestCaseNew {
+
+    @Test
+    public void test_001_AddRandomItemsToCart() {
+        new IndexPageNew(driver)
+                .openSite()
+                .addRandomItemsFromIndexPageToCart(2)
+                .getCartPage();
+
+        assertTrue(new CartPageNew(driver)
```

Comment:
It is important assert, so I think that it is easier to create two different tests, for example